go/private/go_repository.bzl

```diff
@@ -57,6 +57,13 @@ def _new_go_repository_impl(ctx):
   if result.return_code:
     fail("failed to generate BUILD files for %s: %s" % (
         ctx.attr.importpath, result.stderr))
+  ctx.file('WORKSPACE', """
+# Automatically generated by //go/private:go_repository.bzl
+workspace(name = "{workspace_name}")
+{repo_deps}
+""".format(
+    workspace_name = ctx.name,
+    repo_deps="\n".join(["local_repository(name='%s', path='/dummy')" % r for r in ctx.attr.repo_deps])))
```

Comment:
I could add a built-in "dummy" repository rule, if that works better for you guys.